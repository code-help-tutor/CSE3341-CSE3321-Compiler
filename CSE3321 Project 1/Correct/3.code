
program int X , Y , XY 	; a ( b, c, d) begin a=a+b+c; ;begin

          X = 0;
		  
		  output X;

          end  
      
and instance