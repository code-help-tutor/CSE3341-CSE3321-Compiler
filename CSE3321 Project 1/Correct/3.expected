ID[program]
ID[int]
ID[X]
COMMA
ID[Y]
COMMA
ID[XY]
SEMICOLON
ID[a]
LPAREN
ID[b]
COMMA
ID[c]
COMMA
ID[d]
RPAREN
BEGIN
ID[a]
EQUAL
ID[a]
ADD
ID[b]
ADD
ID[c]
SEMICOLON
SEMICOLON
BEGIN
ID[X]
EQUAL
CONST[0]
SEMICOLON
ID[output]
ID[X]
SEMICOLON
END
AND
ID[instance]
